doc/keycmds.dsv

```diff
@@ -68,3 +68,4 @@ home||HOME||Go to the first item in the list.
 end||END||Go to the last item in the list.
 macro-prefix||,||Initiate macro execution. The next key press selects the actual macro and runs it.
 switch-focus||TAB||Switch focus between widgets. This is currently only applicable to the `filebrowser` and `dirbrowser` contexts.
+goto-title||||Go to item whose title contains the specified string.
```

Comment:
I think this should mention that the string is matched case-insensitively.

Reply:
Done: https://github.com/newsboat/newsboat/pull/1249/commits/466bed93102292fcb129416e01d2f1f37cc0f5fd